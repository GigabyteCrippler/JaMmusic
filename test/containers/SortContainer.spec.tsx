import React from 'react';
import { shallow } from 'enzyme';
import Sort from '../../src/containers/SortContainer';

describe('sort container', () => {
  let wrapper: any;
  beforeEach(() => {
    wrapper = shallow(<Sort />);
  });
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
