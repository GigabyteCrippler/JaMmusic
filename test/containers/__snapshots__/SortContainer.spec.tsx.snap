// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sort container renders correctly 1`] = `ShallowWrapper {}`;
